packages/amplify-appsync-simulator/src/velocity/util/info.ts

```diff
@@ -27,7 +27,7 @@ export function createInfo(info: GraphQLResolveInfo) {
   const fieldNode = info.fieldNodes.find(f => f.name.value === info.fieldName);
   if (fieldNode && fieldNode.selectionSet) {
     const query = print(fieldNode);
-    selectionSetGraphQL = query.substr(query.indexOf('{'));
+    selectionSetGraphQL = query.substr(query.indexOf(' {\n') + 1);
```

Comment:
is the leading space and trailing newline always present? Or should it be something like:
`query.search(/\s+{\n?/)`

Reply:
@edwardfoyle Thanks for you feedback
it seems like it is always present. We could use regex though. 👍 

Also, maybe a better option might be to call [visit](https://github.com/graphql/graphql-js/blob/master/src/language/visitor.js#L225)  with a custom or altered [visitor](https://github.com/graphql/graphql-js/blob/master/src/language/printer.js#L11)

I will try to see what I can find out.

Reply:
@edwardfoyle Solution eded up being much simpler than expected. 
See latest change.

Thanks